library/src/trips/index.ts

```diff
@@ -127,8 +128,8 @@ export const searchTrips = (query: string): TripWithScore[] => {
   } = citiesResult;
 
   const distanceInKm = getDistanceFromLatLonInKm(
-    origin.loc.coordinates,
-    destination.loc.coordinates
+    origin.location!,
+    destination.location!
```

Comment:
`CityProto.location` shouldn't be nullable, no?

Reply:
By design, all protocol fields are optional. So they're all nullable. There is no way to change this in the `.proto` file and generated code makes them nullable.